eng/common/scripts/Helpers/DevOps-WorkItem-Helpers.ps1

```diff
@@ -490,21 +545,45 @@ function CreateOrUpdatePackageWorkItem($lang, $pkg, $verMajorMinor, $existingIte
       }
     }
 
-    $newparentItem = FindOrCreatePackageGroupParent $serviceName $pkgDisplayName -outputCommand $false
+    $newparentItem = FindOrCreateReleasePlanParent $serviceName $pkgDisplayName -outputCommand $false
     UpdateWorkItemParent $existingItem $newParentItem -outputCommand $outputCommand
     return $existingItem
   }
 
-  $parentItem = FindOrCreatePackageGroupParent $serviceName $pkgDisplayName -outputCommand $false
+  $parentItem = FindOrCreateReleasePlanParent $serviceName $pkgDisplayName -outputCommand $false
   $workItem = CreateWorkItem $title "Package" "Release" "Release" $fields $assignedTo $parentItem.id -outputCommand $outputCommand
   Write-Host "[$($workItem.id)]$lang - $pkgName($verMajorMinor) - Created"
   return $workItem
 }
 
+function FindOrCreateReleasePlanParent($serviceName, $packageDisplayName, $outputCommand = $true)
+{
+  $existingItem = FindReleasePlanWorkItem $serviceName $packageDisplayName -outputCommand $outputCommand
+  if ($existingItem) {
+    Write-Host "Found existing release plan work item [$($existingItem.id)]"
+    $newparentItem = FindOrCreatePackageGroupParent $serviceName $packageDisplayName -outputCommand $outputCommand
+    UpdateWorkItemParent $existingItem $newParentItem
+    return $existingItem
+  }
+
+  $fields = @()
+  $fields += "`"PackageDisplayName=${packageDisplayName}`""
+  $fields += "`"ServiceName=${serviceName}`""
+  $productParentItem = FindOrCreatePackageGroupParent $serviceName $packageDisplayName -outputCommand $outputCommand
+  $title = "Release Plan - $($packageDisplayName)"
+  $workItem = CreateWorkItem $title "Release Plan" "Release" "Release" $fields $null $productParentItem.id
+
+  $localKey = BuildHashKey $serviceName $packageDisplayName
+  Write-Host "[$($workItem.id)]$localKey - Created release plan work item"
+  $releasePlanWorkItems[$localKey] = $workItem
+  return $workItem
+}
+
 function FindOrCreatePackageGroupParent($serviceName, $packageDisplayName, $outputCommand = $true)
```

Comment:
this function created the product wi right? If so we could updae line 600 so it says "Created product work item" .
Currently it says "Parent" which is confusing

Reply:
I will change it.